README.md

```diff
@@ -271,6 +274,9 @@ database (plus a '.') is used.
 `-v` Specifies verbose mode. This will cause HarbourBridge to output detailed
 messages about the conversion.
 
+`-schema-only` Specifies that conversion would be schema-only and will
+exclude data conversion.
```

Comment:
Suggest rewording:

Specifies that only schema processing will be performed. Any data in the source database
will be ignored.